components/purchase_modal/purchase_modal.tsx

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {Stripe, loadStripe} from '@stripe/stripe-js';
+import {Elements} from '@stripe/react-stripe-js';
+
+import upgradeImage from 'images/cloud/upgrade.svg';
+import wavesBackground from 'images/cloud/waves.svg';
+import blueDotes from 'images/cloud/blue.svg';
+
+import cloudLogo from 'images/cloud/mattermost-cloud.svg';
+
+import RootPortal from 'components/root_portal';
+import FullScreenModal from 'components/widgets/modals/full_screen_modal';
+import {areBillingDetailsValid, BillingDetails} from 'components/cloud/types/sku';
+import {getNextBillingDate} from 'utils/utils';
+
+import PaymentForm from '../payment_form/payment_form';
+
+import './purchase.scss';
+import 'components/payment_form/payment_form.scss';
+import ProcessPaymentSetup from './process_payment_setup';
+
+const STRIPE_CSS_SRC = 'https://fonts.googleapis.com/css?family=Open+Sans:400,400i,600,600i&display=swap';
+const STRIPE_PUBLIC_KEY = 'pk_test_ttEpW6dCHksKyfAFzh6MvgBj';
```

Comment:
@joram this is the same as in CWS. Do we need to change this in the build process for the webapp?

Reply:
(Once again I mentioned wrong, apologies `@joram`!)

@jwilander see above

Reply:
Yes, we'll need to replace this when building releases

Reply:
is this a real key? ahh should this even be here? I propose removing it from the webapp @marianunez